docs/4.0/layout/grid.md

```diff
@@ -515,7 +515,7 @@ In addition to column clearing at responsive breakpoints, you may need to reset
 
 ### Order classes
 
-Use `.order-` classes for controlling the **visual order** of your content. These classes are responsive, so you can set the `order` by breakpoint (e.g., `.order-1.order-md-2`). Includes support for `1` through `12` across all five grid tiers.
+Use `.order-` classes for controlling the **visual order** of your content. These classes are responsive, so you can set the `order` by breakpoint (e.g., `.order-1.order-md-2`). Includes support for `1` through `12` across all five grid tiers. You also set the class of an element to be `first` which will give it negative order and so put it before unordered elements.
```

Comment:
You **can** also set the class of an element to be first with `.order-first`, which will give it a negative order and so place it before the unordered elements.